cmd/network/packet-capture.go

```diff
@@ -517,15 +517,16 @@ func waitForPacketCapturePod(o *packetCaptureOptions, capturePod *corev1.Pod) er
 }
 
 func setCaptureInterface(o *packetCaptureOptions) error {
-	ds := &appsv1.DaemonSet{}
-	if err := o.kubeCli.Get(context.TODO(), types.NamespacedName{Namespace: "openshift-ovn-kubernetes", Name: "ovnkube-master"}, ds); err != nil && !apierrors.IsNotFound(err) {
+	networkConfig := &configv1.Network{}
+	if err := o.kubeCli.Get(context.Background(), client.ObjectKey{Name: "cluster"}, networkConfig); err != nil {
 		return fmt.Errorf("failed to determine the network type: %s", err)
 	}
 
-	if ds.Status.DesiredNumberScheduled > 0 {
+	if networkConfig.Spec.NetworkType == "OVNKubernetes" {
 		o.captureInterface = "genev_sys_6081"
 		return nil
+	} else {
+		o.captureInterface = "vxlan_sys_4789"
+		return nil
 	}
```

Comment:
Does this mean that SDN uses `vxlan_sys_4789` and then OVN obviously uses `genev_sys_6081`? If so, let's make this a case block and then have a `default:` throw and error for an unknown/unsupported network type.